src/components/dashboard/navigation.js

```diff
@@ -52,7 +53,8 @@ class Navigation extends Component {
               className="Groa-NavLink"
               to={`/${this.props.userid}/recommended`}
             >
-              Groa
+              
+              <img src={GroaLogo} alt="Groa Logo" />
```

Comment:
This seems to be underlined as rendering because it links to the component recommended, can this just not be a link since we have one for that anyway? As it stands, both are underlined when the component displays, which is confusing.

Reply:
fixed